main.go

```diff
@@ -133,6 +134,8 @@ func InitFlags(fs *pflag.FlagSet) {
 
 	fs.StringVar(&lbProvider, "lb-provider", "amphora",
 		"The name of the load balancer provider (amphora or ovn) to use (defaults to amphora).")
+
+	fs.StringVar(&cacertsPath, "ca-certs", "", "The path to a ca certificate file to supply as default cacert for each requests.")
```

Comment:
Might be good to document here, if the secret takes precedence over the default path provided here.
```suggestion
	fs.StringVar(&cacertsPath, "ca-certs", "", "The path to a PEM-encoded CA Certificate file to supply as default for each request.")
```